Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN npm config set unsafe-perm true
 RUN npm install -g snyk
 RUN pip install boto3
 RUN apk add --no-cache --virtual=build-dependencies curl wget tar && \
-  curl -sL "https://piccolo.link/sbt-$SBT_VERSION.tgz" | gunzip | tar -x -C /usr/local && \
+  curl -sL "https://github.com/sbt/sbt/releases/download/v$SBT_VERSION/sbt-$SBT_VERSION.tgz" | gunzip | tar -x -C /usr/local && \
```

Comment:
can you please add `-f` or `--fail` to curl so that it fails if it doesn't receive a 200 back?

Reply:
make sense. Done!